.github/workflows/verify.yaml

```diff
@@ -15,12 +15,18 @@ jobs:
 
     services:
       postgres:
-        image: postgres:9.3
+        image: postgres:13.7
```

Comment:
Same thought here

Reply:
I'm planning on reprovisioning a new server that has a supported OS. It's hard to install such new versions of node, which is required by new bower, on 16.04. plus I can save some money with the other #484 but we can't shrink the disk on the existing one. Considering it's already all tested on 20 and 22 I don't think it will be nearly as much work as the rest of this already was.